DecipherNow/gm-fabric-dashboard
# chore: add GMLineChart to Storybook

Resolves #1275 

## Proposed Changes

  - Update how our Dygraph wrapper handles comparison of deep Dygraph options objects
  - Add GMLineChart to Storybook
  
## How to Test

  - Run `npm test`
  - Run storybook and fiddle with the knobs. You should be able to add any options from http://dygraphs.com/options.html
